tests/Checker/AJAX_Runner_Tests.php

```diff
@@ -61,9 +63,14 @@ function( $checks ) {
 			}
 		);
 
+		$muplugins_loaded = $wp_actions['muplugins_loaded'];
+		unset( $wp_actions['muplugins_loaded'] );
```

Comment:
As this same code is repeated, maybe 
1. Create a trait, to reuse code. 
2. Move set_up / tear_down.

Reply:
Thanks @spacedmonkey I agree, but lets tackle this type of refactoring as part of the milestone 1 review once the architecture is complete.

Reply:
This one, I don't agree with. I don't like this how it is. Adding a trait is simple enough, let's just do this one now.

Reply:
Break out issue. https://github.com/10up/plugin-check/issues/109